README.md

```diff
@@ -454,4 +454,4 @@ val value = @@[Width](5) // value is `Int @@ Width`
 - Estatico's [scala-newtype](https://github.com/estatico/scala-newtype) (based on macros, not bad alternative)
 - **Alexander Semenov**'s [Tagged-Types](https://github.com/Treev-io/tagged-types/)
 - [shapeless tags & newtype (classic)](https://github.com/milessabin/shapeless) (very poor)
-- [scalaz tags](https://github.com/scalaz/scalaz/blob/0e75c20aee7634de06fe94094cceac7a3e5fe305/core/src/main/scala/scalaz/Tag.scala) (only tagged, very poor)
\ No newline at end of file
+- [scalaz tags](https://github.com/scalaz/scalaz/blob/0e75c20aee7634de06fe94094cceac7a3e5fe305/core/src/main/scala/scalaz/Tag.scala) (only tagged, very poor)
```

Comment:
I am not sure why this is showing up as a chance. 🤔  I fixed only a typo on line 27.